stubs/opencv-python/cv2/__init__.pyi

```diff
@@ -0,0 +1,21 @@
+from typing_extensions import TypeAlias
+
+from cv2 import (
+    Error as Error,
+    data as data,
+    gapi as gapi,
+    mat_wrapper as mat_wrapper,
+    misc as misc,
+    utils as utils,
+    version as version,
+)
+from cv2.cv2 import *
+from cv2.mat_wrapper import Mat as WrappedMat, _NDArray
```

Comment:
`cv2.WrappedMat` doesn't exist for me (on Mac, 3.10, with opencv 4.5 or 4.6)

Reply:
Should've been a private alias. I'm surprised stubtest didn't catch that.